From 10d79711b61bfc5198d8f89b7338e4468e409e38 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 25 Dec 2019 11:12:38 +0800
Subject: [PATCH] ARM: rockchip_defconfig: enable IOSCHED_BFQ

+CONFIG_IOSCHED_BFQ=y
+CONFIG_BFQ_GROUP_IOSCHED=y

For better app launch time when MTP background copy.
		adobe            aiqiyi        facebook
mq-deadline     12s299ms        1m12s382ms     2m31s691ms
bfq             1s389ms         3s492ms	       1s933ms

Change-Id: I449ba410efe09cfaef2598c853fe1c7b6e53dbf4
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 5cd0772b69fe..6b807761c0a5 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -85,6 +85,8 @@ CONFIG_MODULE_FORCE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 CONFIG_PARTITION_ADVANCED=y
 CONFIG_CFQ_GROUP_IOSCHED=y
+CONFIG_IOSCHED_BFQ=y
+CONFIG_BFQ_GROUP_IOSCHED=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_CMA=y
-- 
2.35.3

